docs/guide/known-limitations.md

```diff
@@ -32,7 +32,7 @@ you can't compare the arguments in a formula like this:
 * SUBTOTAL function does not ignore nested subtotals.
 * CHISQ.INV, CHISQ.INV.RT, CHISQ.DIST.RT, CHIDIST, CHIINV and CHISQ.DIST (CHISQ.DIST in CDF mode): Running time grows linearly with the value of the second parameter, degrees_of_freedom (slow for values>1e7).
 * GAMMA.DIST, GAMMA.INV, GAMMADIST, GAMMAINV (GAMMA.DIST and GAMMADIST in CDF mode): Running time grows linearly with the value of the second parameter, alpha (slow for values>1e7). 
-* RATE function might have no solutions (depending on the inputs). Sometimes it has multiple solutions. Our implementation uses an iterative algorithm (Newton's Method) for finding an approximation for one of them to within `1e-7`. If it is not found after `50` iterations function returns the `#NUM!` error.
+* For certain inputs, the RATE function might have no solutions, or have multiple solutions. In case of multiple solutions, our implementation uses an iterative algorithm (Newton's method) to find an approximation for one of the solutions to within `1e-7`. If the approximation is not found after 50 iterations, the RATE function returns the `#NUM!` error.
```

Comment:
```suggestion
* For certain inputs, the RATE function might have no solutions, or have multiple solutions. Our implementation uses an iterative algorithm (Newton's method) to find an approximation for one of the solutions to within `1e-7`. If the approximation is not found after 50 iterations, the RATE function returns the `#NUM!` error.
```

"In case of multiple solutions" is redundant. The algorithm has no way of knowing if a solution exists upfront, so it tries to find it anyway. If no solution is found after 50 iterations, it throws an error.

Reply:
Thanks! Fixed: [6f7978d](https://github.com/handsontable/hyperformula/pull/962/commits/6f7978d3b57dc9a15cf081f1bfaa6ba7ba9b311d)